### PR TITLE
mark dmidecode and logrotate as optional for Liberty 7 LTSS

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Mark additional packages as optional.
Liberty 7 LTSS for OL do not have all the packages in the repo and generating the bootstrap repo fails for this.
With marking them optional they can be ignored without producing error messages.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Port(s): https://github.com/SUSE/spacewalk/pull/25485

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: susemanager/src/mgr_bootstrap_data.py [hunk 1]
```diff
@@ -103,11 +103,11 @@
 ]
 
 RES7 = [
-    "logrotate",
+    "logrotate*",
```
Comment (agraul) [inline susemanager/src/mgr_bootstrap_data.py:106]: logrotate is not an optional dependency for the Bundle, see e.g. https://build.suse.de/projects/SUSE:Maintenance:35074/packages/venv-salt-minion.SUSE_RES-7_Update_Products_SaltBundle_Update/repositories/SUSE_RES-7_Update_Products_SaltBundle_Update/binaries/x86_64/venv-salt-minion-3006.0-2.58.1.x86_64.rpm

Comment (mcalmer) [reply susemanager/src/mgr_bootstrap_data.py:106]: yes, but RES7 LTSS repo does not provide that package. It result always in build errors.
As I think logrotate is in 99% of all the cases already installed, I would vote for this shortcut.
It only effect RES7 which is LTSS. By the rules no, new installation should happen for LTSS versions.

Comment (mcalmer) [reply susemanager/src/mgr_bootstrap_data.py]: I have changed it and now we have logrotate only optional for `SLL7-OL-LTSS-x86_64` where the default channel does not have this package.

**Review by agraul — APPROVED** [review]
LGTM! :thumbsup:

**Pull request merged**